services/onboarding-service/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/OnboardingService.java

```diff
@@ -1144,6 +1144,16 @@ protected void handleOnboardingProvisioned(Map<String, Object> event, Context co
                             )
                     );
                 }
+
+                // Publish an event to subscribe this tenant to the billing system if needed
+                // TODO Onboarding probably shouldn't be sending the plan in -- just the tier and/or tenant
```

Comment:
This is a good callout. We can address this as part of revamping the billing service.